uber/server.py

```diff
@@ -11,6 +11,32 @@ def _add_email():
 cherrypy.tools.add_email_to_error_page = cherrypy.Tool('after_error_response', _add_email)
 
 
+def _custom_verbose_logger(debug=False):
+    """
+    Write the request headers, and the last error's traceback to the cherrypy error log.
+    Do this all one line so all the information can be collected by external log collectors and easily displayed.
+    """
+
+    page_location = 'Request: ' + cherrypy.request.request_line
+
+    admin_name = AdminAccount.admin_name()
+    admin_txt = 'Current admin user is: {}'.format(admin_name if admin_name else '[non-admin user]')
+
+    max_reporting_length = 1000   # truncate to reasonably large size in case they uploaded attachments
+
+    p = ["  %s: %s" % (k, v[:max_reporting_length]) for k, v in cherrypy.request.params.items()]
```

Comment:
got around to testing this, the output actually (thankfully) doesn't show the actual file contents, it just shows something like:
```
my_file_param:  'strongbad.jpg'
```

so, we're good!